docker-compose.cloud.yaml

```
@@ -0,0 +1,74 @@
+      - caddy_config:/config
+
+volumes:
+  mongodb-data:
+  redis-data:
+  caddy_data:
+  caddy_config:
+  minio-data:
+
+networks:
+  talawa-network:
+    driver: bridge
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)